generator/locale/en/LC_MESSAGES/django.po

```diff
@@ -1067,15 +1067,15 @@ msgid ""
 "CreativeCommons BY-SA pour les tablatures de la base de chansons. Si vous "
 "voulez participer à l'amélioration du projet ou de la base de chansons, "
 "rendez-vous sur <a href=\"http://github.com/patacrep/\">Github</a>, ou sur "
-"le <a href=\"http://www.patacrep.com/forum/\">forum</a> du projet.\n"
+"le <a href=\"http://www.patacrep.fr/forum/\">forum</a> du projet.\n"
 "\t"
 msgstr ""
 "\n"
 "\tThe project code is provided under the GPLv2 license ,the tablautures and "
 "songs are provided under the CreativeCommons BY-SA licences. If you would "
 "like to help imporve the site or the songs database, please go check out our "
 "<a href=\"http://github.com/patacrep/\">Github</a>, or our <a href=\"http://"
-"www.patacrep.com/forum/\">forum</a> for the project.\n"
+"www.patacrep.fr/forum/\">forum</a> for the project.\n"
```

Comment:
Switch to GitHub issues URL (https://github.com/patacrep/patacrep/issues/).

(Can't offer code suggestion here since the `http` need to be changed to `https` but is not part of the diff)